frontend/src/components/DocumentPreview.js

```diff
@@ -43,10 +43,14 @@ export class DocumentPreview extends Component {
     super(props)
 
     this.state = {
-      accessToken: null
+      modal: false,
```

Comment:
modal never was initialized before but since null is false i guess it worked out